### DISCUSSION TITLE
hi this is my ipxe menu  I want to install ubuntu 22.04 desktop

### DISCUSSION THREAD

**Discussion by qsc811022** [ipxe/ipxe#674] [Q&A] [answered]
:shell
shell
:linux-net1
kernel http://192.168.0.128/ubuntu/casper/vmlinuz inst.repo=http://192.168.0.128/ubuntu initrd=initrd
initrd http://192.168.0.128/ubuntu/casper/initrd
boot

will result in the following
How can I change the menu so that I can get to the installation screen smoothly

![image](https://user-images.githubusercontent.com/11502486/178005500-2b591566-0c40-47c5-81d0-bc0540ed85d4.png)

Please add this text

#=================================================================================================#
:shell
shell
#/var/www/html/ubuntu22.04
#desktop
#ubuntu-22.04-desktop-amd64.iso
#ubuntu22.04/casper

:linux-net5
set server_root http://192.168.3.1
kernel ${server_root}/ubuntu22.04/casper/vmlinuz 
initrd ${server_root}/ubuntu22.04/casper/initrd
imgargs vmlinuz initrd=initrd root=/dev/ram0 ramdisk_size=3800000 ip=dhcp url=http://192.168.3.1/ubuntu-22.04-desktop-amd64.iso
#locale=en_US.UTF-8 ro
boot

#=================================================================================================#

:shell
shell
#/var/www/html/ubuntu22.04
#Server
#/var/www/html/ubuntuServer22.04
#ubuntu-22.04-live-server-amd64.iso

:linux-net6
set server_root http://192.168.3.1
kernel ${server_root}/ubuntuServer22.04/casper/vmlinuz 
initrd ${server_root}/ubuntuServer22.04/casper/initrd
imgargs vmlinuz initrd=initrd root=/dev/ram0 ramdisk_size=1600000 ip=dhcp url=http://192.168.3.1/ubuntu-22.04-live-server-amd64.iso
#locale=en_US.UTF-8 ro
boot

#=================================================================================================#

**Comment by stappersg** [comment]
[This is a duplicate of #671](https://giveupgithub.org/)

**Reply by chelmuth** [reply]
Hiding links to hacktivism pages behind innocent discussion posts is not very professional.

**Reply by stappersg** [reply]
The link that indeed shouldn't have been hidden:  https://giveupgithub.org/

**Comment by stappersg** [comment, marked as answer]
@qsc811022 Please add this text
```text
#=================================================================================================#
:shell
shell
#/var/www/html/ubuntu22.04
#desktop
#ubuntu-22.04-desktop-amd64.iso
#ubuntu22.04/casper

:linux-net5
set server_root http://192.168.3.1
kernel ${server_root}/ubuntu22.04/casper/vmlinuz 
initrd ${server_root}/ubuntu22.04/casper/initrd
imgargs vmlinuz initrd=initrd root=/dev/ram0 ramdisk_size=3800000 ip=dhcp url=http://192.168.3.1/ubuntu-22.04-desktop-amd64.iso
#locale=en_US.UTF-8 ro
boot

#=================================================================================================#

:shell
shell
#/var/www/html/ubuntu22.04
#Server
#/var/www/html/ubuntuServer22.04
#ubuntu-22.04-live-server-amd64.iso

:linux-net6
set server_root http://192.168.3.1
kernel ${server_root}/ubuntuServer22.04/casper/vmlinuz 
initrd ${server_root}/ubuntuServer22.04/casper/initrd
imgargs vmlinuz initrd=initrd root=/dev/ram0 ramdisk_size=1600000 ip=dhcp url=http://192.168.3.1/ubuntu-22.04-live-server-amd64.iso
#locale=en_US.UTF-8 ro
boot

#=================================================================================================#
```
to #671 so the duplicate question also gets a duplicated answer. And please, pretty please, keep the _give and take_ in balance.
